factfinder/geography/2010_to_2020.py

```diff
@@ -60,6 +58,17 @@ def create_output(df, colname):
     def agg_moe(x):
         return math.sqrt(sum([i ** 2 for i in x]))
 
+    @staticmethod
+    def convert_moe(e_2010, m_2010, e_2020, ratio):
+        if ratio == 1:
+            return m_2010
+        elif e_2020 == 0:
+            return None
+        elif (ratio**(0.56901))*7.96309 >= 100:
```

Comment:
Should it be `(ratio**(0.56901))*7.96309 >= 1`? I don't see how this number (`(ratio**(0.56901))*7.96309`) can get that large

Reply:
I agree. Let's get Population sign off.